### PR TITLE
12 enforce conventional commits and code styles

### PR DESCRIPTION
**Pull request by tibohei**
merge #12

### CODE REVIEW

File: src/mappings/orders.ts [hunk 1]
```diff
@@ -17,13 +17,13 @@ async function _handleRedeemOrderUpdated(event: SubstrateEvent): Promise<void> {
 
 const handleOrderUpdated = errorHandler(_handleOrderUpdate)
 async function _handleOrderUpdate(event: SubstrateEvent, type: InvestorTransactionType) {
-  const [poolId, trancheId, _address, _oldOrder, newOrder] = event.event.data as unknown as OrderEvent
+  const [poolId, trancheId, newOrder] = event.event.data as unknown as OrderEvent
```
Comment (tibohei) [inline src/mappings/orders.ts:20]:
```suggestion
  const [poolId, trancheId, , , newOrder] = event.event.data as unknown as OrderEvent
```

Comment (tibohei) [inline src/mappings/orders.ts:20]:
```suggestion
  const [poolId, trancheId, , , newOrder] = event.event.data as unknown as OrderEvent
```

Comment (tibohei) [inline src/mappings/orders.ts:20]:
```suggestion
  const [poolId, trancheId, , , newOrder] = event.event.data as unknown as OrderEvent
```

**Review by filo87 — APPROVED** [review]
LG Thx

**Pull request merged**